media/admin/com_virtuemart/views/zasilkovna/tmpl/order_extended_detail.php

```diff
@@ -40,6 +40,39 @@
             <th class="key va-middle"><?php echo JText::_('PLG_VMSHIPMENT_PACKETERY_PACKET_PRICE'); ?></th>
             <td><?php echo htmlentities($order->getZasilkovnaPacketPrice()); ?></td>
         </tr>
+        <?php if ($order->getLength()) {
```

Comment:
značky `<?php` a `?>` bych dával vždy na samostatný řádek

Reply:
takhle jsem to myslel, je to jen o 2 řádky delší a přehlednější. ještě bych aplikoval formátování